src/CMakeLists.txt

```diff
@@ -11,6 +11,8 @@ add_library(linuxdeploy_desktopfile STATIC
     desktopfilewriter.h
     util.h
     ${HEADERS}
-)
+    )
 
 target_include_directories(linuxdeploy_desktopfile PUBLIC ${PROJECT_SOURCE_DIR}/include)
+
+set_target_properties(linuxdeploy_desktopfile PROPERTIES POSITION_INDEPENDENT_CODE ON)
```

Comment:
How about setting this globally for the entire project (as in, `set(CMAKE_POSITION_INDEPENDENT_CODE ON) in the root `CMakeLists.txt`)? I doubt that this applies the property to all dependencies.

Reply:
This only works because we have just one library right now. But what if we decide to add more in the future? I have doubts anyone will remember to enable that property. I'd appreciate if you enabled it project-globally instead.

Reply:
Done.